apps/api/v2/src/modules/organizations/controllers/memberships/organizations-membership.controller.ts

```diff
@@ -0,0 +1,124 @@
+import { API_VERSIONS_VALUES } from "@/lib/api-versions";
```

Comment:
Would be good to add e2e tests to the controller - we try to have each controller covered. For example can see `controllers/organizations-teams.controller.e2e-spec.ts`.